virttest/qemu_storage.py

```diff
@@ -1096,6 +1096,128 @@ def snapshot_apply(self):
 
         process.run(cmd)
 
+    def bitmap_add(self):
+        """
+        Add a bitmap to a image.
+
+        :note: params should contain:
+               bitmap_name -- the name of bitmap
+        """
+
+        cmd = self.image_cmd
+        if self.image_format == 'qcow2':
+            cmd += " bitmap --add -g %s -f %s %s %s" % (
```

Comment:
bitmap_params = self.params.object_params(bitmap_name)
granularity = bitmap_params.get('bitmap_granularity')
if granularity:
    cmd += ' -g %s' % granularity
granularity is not a must when adding a bitmap